### ISSUE TITLE
Installation of library for Linux/MacOS

### ISSUE THREAD

**Issue by ihhub** [ihhub/penguinV#47]
To simplify developer's life it would be useful to investigate the possibility of installation library within OS so it would be stored in one place.

**Comment by 0x72D0** [comment]
on my manjaro distribution, the library are usually install under /usr/lib, the header file under /usr/include and the documentation under /usr/share. Maybe I can try to do an AUR package of this project for arch linux.

**Comment by ihhub** [comment]
Yeah, it would be nice. We're going forward to make a release. As of now we need to merge together CPU, CUDA and OpenCL code under single package.
